package.json

```diff
@@ -7,20 +7,22 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "test": "playwright test"
+    "test": "playwright test",
+    "prettier": "prettier --write ."
```

Comment:
I suggest we only format the files which are tracked in the repository by changing this to `"prettier --write --ignore-path .gitignore ."`.

Reply:
made a commit with the latest changes based on these suggestions